default-compiler-clis/BUILD.bazel

```diff
@@ -0,0 +1,34 @@
+java_binary(
```

Comment:
We could do something like this
```
scala_play_version = [
    ["2_11", "2_5"],
    ["2_11", "2_6"],
    ["2_11", "2_7"],
    ["2_12", "2_6"],
    ["2_12", "2_7"]
]
[
    java_binary(
      name = "scala_{}_play_{}".format(scala_version, play_version),
      runtime_deps = ["@play_routes//:com_lucidchart_play_routes_compiler_cli_{}".format(scala_version)],
      main_class = "rulesplayroutes.routes.CommandLinePlayRoutesCompiler",
      visibility = ["//visibility:public"]
    )
    for scala_version, play_version in scala_play_version
]
```
You would also want to verify the syntax.

Reply:
I don't like the idea of using `for` loops in BUILD files.. the fact that it needs to be in a for comprehension to get around `for` loops not being allowed at the top level in BUILD files feels like a red flag

Reply:
I do like that it's much more concise... On the other hand, however, this file will not be changing frequently